pkg/utils/constants.go

```diff
@@ -19,5 +19,7 @@ const (
 	LMSPort = "16992"
 
 	// MPSServerMaxLength is the max length of the servername
-	MPSServerMaxLength = 256
+	MPSServerMaxLength    = 256
+	ReturnCode_SUCCESS    = 0
```

Comment:
Nice that these are constants. Can you update the names to be just SUCCESS or FAIL, in any case -- remove the underscores and use mixedCase.

Reply:
done